src/main/java/com/locallampoon/fiveh/ui/mappanel/GameMapPanel.java

```diff
@@ -44,7 +45,6 @@ public void draw(Graphics graph){
             } else
                 this.drawRoom(graph, r.getValue(), spotLight, neighbour);
         }
-        this.drawPlayer(graph);
```

Comment:
what's the intention behind removing this?

Reply:
drawPlayer leaves a dot in the upper left corner. That is an artifact from Xander's original animated player implementation, I believe.

Reply:
I removed from my current branch too. not a big deal now, since we are not using that poor "player"